Fix system not loading due to:

Unhandled CFFI:LOAD-FOREIGN-LIBRARY-ERROR 
  Unable to load foreign library (LIBALLEGRO-NUKLEAR).
  Error opening shared object "/build/source/src/liballegro_nuklear.so":
  /build/source/src/liballegro_nuklear.so: undefined symbol: al_draw_ellipse.
--- a/cl-liballegro-nuklear.asd
+++ b/cl-liballegro-nuklear.asd
@@ -12,7 +12,7 @@
   :description "CFFI wrapper for the Nuklear IM GUI library with liballegro backend, to be used with cl-liballegro."
   :author "Andrew Kravchuk <awkravchuk@gmail.com>"
   :license "MIT"
-  :depends-on (:cffi :cffi-libffi :trivial-features)
+  :depends-on (:cl-liballegro :cffi :cffi-libffi :trivial-features)
   :pathname "src"
   :serial t
   :components ((:makefile "Makefile")
